subcommand/server-acl-init/command_test.go

```diff
@@ -392,7 +310,7 @@ func TestRun_TokensReplicatedDC(t *testing.T) {
 			tokenFile, fileCleanup := writeTempFile(t, bootToken)
```

Comment:
Above here we have a lot of `SecretName: "release-name-consul-client-snapshot-agent-acl-token"` but I think it should be `SecretName: resourcePrefix + "-client-snapshot-agent-acl-token"`